models/contacts.js

```diff
@@ -1,75 +1,65 @@
-const Joi = require("joi");
-const fs = require("fs/promises");
-const path = require("path");
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
 
-const contactsPath = path.join(__dirname, "contacts.json");
+const connectionString =
+  "mongodb+srv://toysan3:BvNf%253Rimongodb@cluster0.nw4xzpi.mongodb.net/";
```

Comment:
Schowaj to do .env a następnie zrób import. Możesz dostać informacje z mongo atlas że ujawniłeś dane wrażliwe i będziesz musiał zmienić hasło.

Reply:
Zrobione